tests/gpt.rs

```diff
@@ -128,6 +128,45 @@ fn test_create_simple_on_device() {
     mem_device.read_exact(&mut final_bytes).unwrap();
 }
 
+
+#[test]
+fn test_create_tiny_partitions_on_device() {
```

Comment:
It would be great to include a test here for the actual limit. The calculation for the smallest possible set of partitions (including a PMBR should look like):
```
return 1 + // PMBR
    2 + // Primary and backup headers
    ((2 * 2.pow(14)) / lb_size_in_bytes); // The EFI spec provides the minimum 
```

Testing against partitions at this size brought up issues. But significantly more worrying was I was able to configure and write a GptDisk with _well_ under the minimum limit of this size.